runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -69,6 +73,11 @@
 @Slf4j
 public class FishingPlugin extends Plugin
 {
+	private static final int TRAWLER_SHIP_REGION_NORMAL = 7499;
+	private static final int TRAWLER_SHIP_REGION_SINKING = 8011;
+
+	private static final int TRAWLER_ACTIVITY_THRESHOLD = Math.round(0.15f * 255); // 15% of 255
```

Comment:
Is the comment necessary? I suppose this could be made even more explicitly clear without comments by making `255` a constant as well.

Reply:
I guess I could take it out? Was just a note for anyone not knowing offhand what the max value of the activity varbit is

Reply:
I guess that should be a comment on the varbit definition then :P

Reply:
done